demo/App.js

```diff
@@ -26,14 +26,16 @@ function App() {
     {
         content: 'double click this item',
         actionType: 'dblclick',
-        position: 'top',
+        position: 'right',
+        arrowPosition: 'left',
```

Comment:
Can't arrow position be anticipated from `position` prop?

Reply:
position can contain either string or array of number so we need extra variable to get arrow position
New